core/pom.xml

```diff
@@ -35,6 +35,11 @@
   <url>http://spark.apache.org/</url>
   <dependencies>
     <dependency>
+      <groupId>org.apache.spark</groupId>
```

Comment:
I moved the `SparkException` from `core` to `sketch` so I can use `SparkException` in `mllib-local`.